jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/utils/TensorUtils.scala

```diff
@@ -0,0 +1,18 @@
+package ml4ir.inference.tensorflow.utils
+
+import org.tensorflow.Tensor
+
+/**
+  * Simple helpers for creating TensorFlow {@see Tensor}s of the right shapes.
+  */
+object TensorUtils {
+  def replicate(encoding: Array[Float], length: Int): Array[Array[Float]] =
+    Array.ofDim[Float](length, encoding.length).map(_ => encoding.clone())
```

Comment:
`Array.fill` may be cleaner?

Reply:
deleting that class but yes, good point.